projects/js-packages/components/changelog/add-info-outline-gridicon

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Gridicon: added info-outline gridicon to the available subset of icons
```

Comment:
This looks like an extraneous changelog? I don't see any modification to the components package in this PR.

This changelog is causing a minor version bump for `jetpack-components` and therefore triggering a cascade of version bumps and changelogs across the repository.

Reply:
Ah. I wondered why fixup-project-versions created such a cascade of files.

The changelog must be left over from a previous rebase. I'll remove it 👍